src/components/ResultList/Item/Item.tsx

```diff
@@ -177,13 +179,13 @@ const Highlights = ({
   }, [highlights]);
 
   return (
-    <Box as="section" aria-label="highlights" className="search-snippets" my={2} data-testid="highlights-section">
+    <Box as="section" aria-label="highlights" fontStyle="italics" my={2} data-testid="highlights-section">
```

Comment:
The highlighting color is gone, it's hard to tell where the highlights are

Reply:
Good catch, fixed! :+1: